scripts/build-upload-a-docker-image.sh

```diff
@@ -71,3 +71,6 @@ docker buildx build --output "${PUSHTAG}" \
 echo "Finished building${upload_flag} ${component_name} =============="
 
 df -h /
+docker buildx prune --all --verbose
+docker system prune
```

Comment:
We might need to add `--force` for each of these to avoid being prompted for confirmation.